subjects/cybersecurity/image-inspector/audit/README.md

```diff
@@ -0,0 +1,96 @@
+#### General
+
+##### Check the Repo Content
+
+Files that must be inside the repository:
+
+- Detailed documentation in the `README.md` file.
+- Source code for the Image Inspector tool.
+- Any required configuration files and scripts for running the tool.
+
+###### Are all the required files present?
+
+##### Play the Role of a Stakeholder
+
+Organize a simulated scenario where the student takes on the role of a Digital Forensics Expert and explains their solution and knowledge to a team or stakeholder. Evaluate their grasp of the concepts and technologies used in the project, their communication efficacy, and their critical thinking about their solution and the knowledge behind this project.
+
+Suggested role play questions include:
+
+- What is metadata in the context of digital images, and why is it important?
+- How does steganography work, and what are its potential uses and risks?
+- What challenges did you face while developing the Image Inspector tool, and how did you address them?
+- How does the Image Inspector tool help in identifying hidden information within images?
```

Comment:
I don't really find a reply to this question the image inspector tool will help to extract the data then we will do something with the data, maybe you can ask how they can exploit the extracted data ?
Or maybe you can reformulate the question to make it more answerable

Reply:
fixed